controllers/provider-aws/pkg/imagevector/imagevector.go

```diff
@@ -23,11 +23,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/runtime"
 )
 
-// ImageVector is the image vector that contains al the needed images
-var ImageVector imagevector.ImageVector
-
-func init() {
-	var err error
-	ImageVector, err = imagevector.ReadFile(filepath.Join(aws.ChartsPath, "images.yaml"))
+// GetImageVector returns the image vector that contains al the needed images.
+func GetImageVector() imagevector.ImageVector {
+	imageVector, err := imagevector.ReadFile(filepath.Join(aws.ChartsPath, "images.yaml"))
```

Comment:
Please use `packr` as in the other controllers.

Reply:
This is not my code - it was like this before. I only changed the global variable to a function because reading a file from a hardcode path in an `init` function fails during unit tests (they are not executed at the root and the file is not found)

Reply:
You started to adapt the code thus you have to change it as well.

Reply:
I **have to** change it? No, I don't think so.

Reply:
Ok, you should :wink: